libmamba/src/core/run.cpp

```diff
@@ -391,7 +391,7 @@ namespace mamba
             if (ec)
             {
                 std::cerr << ec.message() << '\n';
-                return 1;
```

Comment:
Not sure about this.

Reply:
sorry, I reverted this change. I didn't get the motivation. Is that what conda does?